NLSY/make_nlsy.R

```diff
@@ -1,32 +1,31 @@
 
-# Execute the script provided by NLSY that reads the data and 
+# Execute the script provided by NLSY that reads the data and
 # save them in an RDS file.
 
 library(tidyverse)
 
-# Get the NLSY data path
-source('common.R')
-
-# The NLSY script has to be executed in the data folder
-setwd(nlsy_data_dir)
-source("College-finance2.R")
+# Load in College-finance2.dat
+new_data <- read.table('https://urbanorg.box.com/shared/static/f8tyxcohac1ha11a3mcw0dzq8ywqayyn.dat', sep=' ')
+# Source College-finance2.R to clean the data just loaded in.
+# NOTE: Will need to comment out the first line of this script and resave to Box each time it is updated.
```

Comment:
Flagging that if this code is used in our final analysis, we should make sure to clearly give download instructions so that this work is better reproducible - normally only data would be stored on Box